datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1520,7 +1525,13 @@ type DatasetDeprecation {
 """
 Institutional memory metadata, meaning internal links and pointers related to an Entity
 """
-type InstitutionalMemory {
+type InstitutionalMemory implements Aspect {
+    """
+    The logical version of the schema metadata, where zero represents the latest version
```

Comment:
So each aspect will have its own "semantic version"? 

Will this be confusing given that we also talk about "real aspect versions" all over the place? I'm a bit nervous about this...